### PR TITLE
Add documentation for to-do list card

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Add documentation for to-do list card
- Add documentation for to-do list card
- add links to to-do list card docs from related integrations

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/caldav.markdown [hunk 1]
```diff
@@ -203,3 +203,14 @@ to generate a new password for use by Home Assistant to avoid sharing your iClou
...
 
 {% enddetails %}
+
+## Related topics
+
+- [Calendar](/integrations/calendar)
+
+### To-do list
+
+- [Bring!](https://www.getbring.com/)
```
Comment (allenporter) [inline source/_integrations/caldav.markdown]: Omit this one?

```suggestion
```

**Review by allenporter — DISMISSED** [review]
Looks good, one small suggestion

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_integrations/shopping_list.markdown [hunk 2]
```diff
@@ -16,7 +16,7 @@ ha_platforms:
 
 The `shopping_list` integration allows you to keep track of shopping list items. 
 
-Your shopping list will be accessible from the sidebar, and you can optionally add the [shopping list card](/dashboards/shopping-list/) to your dashboard. With the [conversation integration](/integrations/conversation/) you can add items to your shopping list using voice commands like "Add eggs to my shopping list." 
+Your shopping list will be accessible from the sidebar, and you can optionally add the [Todo list card](/dashboards/todo-list/) to your dashboard. With the [conversation integration](/integrations/conversation/) you can add items to your shopping list using voice commands like "Add eggs to my shopping list."
```
Comment (frenck) [inline source/_integrations/shopping_list.markdown]:
```suggestion
Your shopping list will be accessible from the sidebar, and you can optionally add the [To-do list card](/dashboards/todo-list/) to your dashboard. With the [conversation integration](/integrations/conversation/), you can add items to your shopping list using voice commands like "Add eggs to my shopping list." 
```

File: source/_integrations/todo.markdown [hunk 3]
```diff
@@ -144,4 +144,4 @@ target:
 ## Related topics
 
 - [Local to-do integration](/integrations/local_todo/)
-- [Shopping list card](/dashboards/shopping-list/)
\ No newline at end of file
+- [Todo list card](/dashboards/todo-list/)
```
Comment (frenck) [inline source/_integrations/todo.markdown]:
```suggestion
- [To-do list card](/dashboards/todo-list/)
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**